CMakeLists.txt

```diff
@@ -24,11 +24,17 @@ add_executable(rimage
 	tomlc99/toml.c
 )
 
+set_property(TARGET rimage PROPERTY C_STANDARD 99)
+
 target_compile_options(rimage PRIVATE
 	-Wall -Werror -Wmissing-prototypes -Wimplicit-fallthrough
 )
 
-target_link_libraries(rimage PRIVATE "-lcrypto")
+if(CMAKE_C_COMPILER_VERSION VERSION_GREATER 9.1)
```

Comment:
I think @aborisovich was planning to split the Linux and Windows parts into separate Cmakefile.txt that would be included by the main CMakefile. This would also mean a clang Cmakefile too.

Reply:
you have compiler id on both windows on linux, its agnostic

Reply:
sorry, I though we had different ID strings both linux/windows xcc compiler as the compiler "-linux" suffix is added to the manifest ?

Reply:
@lgirdwood this will be done but not in this PR. It will be [here](https://github.com/intel-innersource/drivers.audio.firmware.converged/pull/30).

Reply:
Ok thanks guys - sorry for the confusion.